core/utils/parsing.ts

```diff
@@ -21,8 +21,8 @@ import * as colourUtils from './colour.js';
  *
  * @param message Text which might contain string table references and
  *     interpolation tokens.
- * @param parseInterpolationTokens Option to parse numeric
```

Comment:
Could we change this param to have a more descriptive name, given the new functionality? My  understanding is that if this is false, then we just parse message references and don't do anything else. So could we use `onlyReplaceMessageReferences = true`?

Reply:
The name onlyReplaceMessageReferences sounds like it would be inverting the current value of the parameter, in which case we'd have to find all the callers and change the argument they provide for the second parameter, which I guess we could do if this is a private API? Another thing we could do it just add a new third parameter called something like "tokenizeNewlines" which defaults to have the same value as the existing "parseInterpolationTokens" parameter.

Reply:
This is indeed module-private (since it's not exported) so we can do whatever we want with it =) I think either solution works here!

Reply:
I added a separate parameter "tokenizeNewlines" to be explicit.